packages/tx-construction/src/createTransactionInternals.ts

```diff
@@ -51,3 +74,8 @@ export const createTransactionInternals = async ({
     )
   };
 };
+
+export const createTransactionInternals = async (props: CreateTxInternalsProps): Promise<Cardano.TxBodyWithHash> => {
```

Comment:
This is declared as an `async function`, but it seems to be sync... maybe it's just to not change its interface?

Reply:
It is async because ti returns the result of `includeChangeAndInputs`, which is a Promise.

Reply:
Done here: [b1a7f4d](https://github.com/input-output-hk/cardano-js-sdk/pull/1099/commits/b1a7f4de95cf58257f625ea9f49fc9f385d25382)